continuous_queries.md

```diff
@@ -0,0 +1,236 @@
+# Continuous Queries
+
+This document lays out continuous queries and a proposed architecture for how they'll work within an InfluxDB cluster.
+
+## Definition of Continuous Queries
+
+Continuous queries serve two purposes in InfluxDB:
+
+1. Combining many series into a single series (i.e. removing 1 or more tag dimensions to make queries more efficient)
+2. Aggregating and downsampling series
+
+The purpose of both types of continuous queries is to duplicate or downsample data automatically in the background to make querying thier results fast and efficient. Think of them as another way to create indexes on data.
```

Comment:
Do you think "index" is a good way to think about this stuff? After all, the data created by a CQ has no direct reference back to the original data, nor makes it quicker to find the original data. Or am I missing something?